project/AppDependencies.scala

```diff
@@ -7,21 +7,21 @@ object AppDependencies {
 
   val compile = Seq(
     "uk.gov.hmrc"   %% "bootstrap-backend-play-30" % BootstrapPlayVersion,
```

Comment:
What of [8.6.0](https://github.com/hmrc/bootstrap-play/releases/tag/v8.6.0)? It's not terribly important and does carry a little risk to logging because I don't understand it yet.